contrib/arrow-cmake/CMakeLists.txt

```diff
@@ -27,11 +64,10 @@ set(ORC_SOURCE_SRC_DIR "${ORC_SOURCE_DIR}/src")
 set(ORC_BUILD_SRC_DIR "${CMAKE_CURRENT_BINARY_DIR}/../orc/c++/src")
 set(ORC_BUILD_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/../orc/c++/include")
```

Comment:
Maybe also separate the orc module?

Reply:
Maybe latter (this PR touches too many files already, and if it will not be merged soon it will have conflicts again and again)